pyodide_build/buildpkg.py

```diff
@@ -58,21 +58,25 @@ def download_and_extract(buildpath, packagedir, pkg, args):
     return srcpath
 
 
-def patch(path, srcpath, pkg, args):
+def patch(path, srcpath, pkg, args, only_extras=False):
     if (srcpath / '.patched').is_file():
-        return
+        if not only_extras:
+            return
+        else:
+            os.remove(srcpath / '.patched')
 
-    # Apply all of the patches
     orig_dir = Path.cwd()
     pkgdir = path.parent.resolve()
     os.chdir(srcpath)
```

Comment:
This is failing in the tests.  We should get to the bottom of why that is, as well.